tests/userTest.js

```diff
@@ -302,7 +362,7 @@ describe('Main: Currently testing user management,', function () {
             ocs.appendElement('meta', '', (meta) => {
               return ocsMeta(meta, 'ok', '100')
             }).appendElement('data', '', data => {
-              data.appendElement('element', '', config.testGroup)
+              data.appendElement('element', '', MatchersV3.string(config.testGroup))
```

Comment:
why do we need this matcher? the group name should be the same on the mock server and on the real server, so we should be able to test it 1:1
similar for a lot of other matchers